workshop/exercise-0/README.md

```diff
@@ -1,7 +1,10 @@
-## Create IBM Cloud Account and Get Cluster
+## Create IBM Cloud Account and Install Developer Tools
 
 If you already have `ibmcloud` installed with the `ibmcloud ks` plugin, you can skip these steps.
```

Comment:
you can need to reword this, or do a bulleted list, since you now have kubctl and knctl in there.  Should add "cr" to the list, and instructions, too.

Reply:
just noticed "cr install" is in e2, let's do it here since the lab install will have it automatically, no reason to split it out from the generic install install section

Reply:
do we know that the lab install will have cr plugin?

Reply:
"Install Developer Tools" was supposed to cover knctl, kubectl, etc.